firestore/angular-rewrite/firestore.rules

```diff
@@ -2,17 +2,14 @@ service cloud.firestore {
   match /databases/{database}/documents {
     match /restaurants/{restaurant} {
       match /ratings/{rating} {
-        allow read: if request.auth != null;
-        allow write: if request.auth.uid == request.resource.data.userId;
+        // Allow anyone to read, but only auth'd users to write
+        allow read: if request.time > timestamp.date(2020, 1, 1);
+        allow write: if request.auth != null;
       }
 
-      allow read: if request.auth != null;
+      allow read: if request.time > timestamp.date(2020, 1, 1);
       allow create: if request.auth != null;
       allow update: if request.auth != null
```

Comment:
We removed the request validation on update - do we not need this anymore?

Reply:
Nope; We don't need the `update` rule anymore! Removing in an upcoming commit!